core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -108,12 +104,15 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.setFirstState = async function(stateName) {
-    await initialStateSelectOption(stateName).click();
+    await action.click(
+      'Initial State Select Option', initialStateSelectOption(stateName));
   };
 
   this.setLanguage = async function(language) {
-    await element(by.css('.protractor-test-exploration-language-select')).
-      element(by.cssContainingText('option', language)).click();
+    await action.click(
+      'Exploration Language Input option',
```

Comment:
Similarly, can we include the language name in the message here?

Reply:
Done